File: pgm/ABC_Dart/ABC_417_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [int N, int A, int B] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  final String S = stdin.readLineSync()!;
  print(S.substring(A, N - B));
  exit(0);
}

File: pgm/ABC_C#/ABC_417_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const int one = 1;
        const int two = 2;
        int A = int.Parse(arr[one]);
        int B = int.Parse(arr[two]);
        String S = Console.ReadLine();
        Console.WriteLine(S.Substring(A, S.Length - A - B));
        Environment.Exit(0);
    }
}